csharp/ScheduledEventsSample/ScheduledEventApproval.cs

```diff
@@ -0,0 +1,36 @@
+﻿/* Copyright 2014 Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+*/
+
+namespace ScheduledEventsSample
+{
+    using System.Collections.Generic;
+    using Newtonsoft.Json;
+
+    public class ScheduledEventsApproval
+    {
+        public List<StartRequest> StartRequests = new List<StartRequest>();
+    }
+
+    public class StartRequest
+    {
+        [JsonProperty("EventId")]
```

Comment:
I might recommend DataMember(name) instead of JSONProperty. [DataMember can be read by Newtonsoft and dotnet's json serializer](http://www.newtonsoft.com/json/help/html/DataContractAndDataMember.htm)